### DISCUSSION TITLE
503 - Request failed with status code 503

### DISCUSSION THREAD

**Discussion by proskater123789** [KelvinTegelaar/CIPP#1712] [Q&A]
I recently installed CyberDrain, but i keep running into this error when I try to load pages. 
![image](https://github.com/KelvinTegelaar/CIPP/assets/124924290/fbf91e80-d9cf-4049-a381-de72f662d653)
It was working just as it should for about 5 mins, and then started getting this error. i can't run a permission check as it pops up a bunch of errors. 
![image](https://github.com/KelvinTegelaar/CIPP/assets/124924290/65cca0c4-6872-4a9a-bbf4-25cf1aba6177)

I have reinstalled the entire app twice hoping I screwed something up during installation, and I keep running into this issue. Could you point me in the right direction of where to start diagnosing this issue?

**Comment by KelvinTegelaar** [comment]
This isn't the place for support. Only sponsors receive formal support.

**Reply by proskater123789** [reply]
So what is the place for support?

**Reply by KelvinTegelaar** [reply]
When you're a sponsornyou receive our support email.
